### ISSUE TITLE
Wrong "List item description must start with valid casing  awesome/list-item" error

### ISSUE THREAD

**Issue by maximveksler** [sindresorhus/awesome-lint#52]
Data

```
cat README.md
- [iqvoc](https://github.com/innoq/iqvoc) - SKOS(-XL) Vocabulary Management System for the Semantic Web.
```

Reproduction 

```
wget https://gist.githubusercontent.com/maximveksler/e04db4c4e50b5756f8865a836028bacf/raw/6fdc20ef697e2e2445e1c52a14012c1f32797344/README.md
Maxims-MBP:lint-bug maximveksler$ awesome-lint
✖ Linting

  README.md:1:1
  ✖  1:1   Missing or invalid Table of Contents                awesome/toc
  ✖  1:1   Missing License section                             awesome/license
  ✖  1:1   Awesome list must reside in a valid git repository  awesome/git-repo-age
  ✖  1:1   Missing file contributing.md                        awesome/contributing
  ✖  1:1   Missing newline character at end of file            final-newline
  ✖  1:42  List item description must start with valid casing  awesome/list-item

  6 errors
```

Live at https://github.com/semantalytics/awesome-semantic-web

Please note that no error is reported for the following example:

```
Maxims-MBP:lint-bug maximveksler$ cat README.md
- [iqvoc](https://github.com/innoq/iqvoc) - SKOS Vocabulary Management System for the Semantic Web.
```

**Comment by ironcev** [comment]
Another example:

    [Bridge](https://github.com/bridgedotnet/Bridge) - C# to JavaScript transpiler. Write modern mobile and web apps in C# and run them anywhere in JavaScript.

To reproduce, run: `
awesome-lint https://github.com/ironcev/awesome-roslyn`

The error is not reported if the `#` character is missing:

    [Bridge](https://github.com/bridgedotnet/Bridge) - C to JavaScript transpiler. Write modern mobile and web apps in C# and run them anywhere in JavaScript.

**Comment by IssuehuntBot** [comment]
@issuehunt has funded $40.00 to this issue.

---
- Submit pull request via [IssueHunt](https://issuehunt.io/repos/64659785/issues/52) to receive this reward.
- Want to contribute? Chip in to this issue via [IssueHunt](https://issuehunt.io/repos/64659785/issues/52).
- Checkout the [IssueHunt Issue Explorer](https://issuehunt.io/issues) to see more funded issues.
- Need help from developers? [Add your repository](https://issuehunt.io/users/44827199/repositories) on IssueHunt to raise funds.

**Comment by chinesedfan** [comment]
I found interesting results of the package [Case](https://github.com/nbubna/Case), which was used to verify list items. So the simple solution is adding upper as a valid case. See #63.

```
> c = require('Case')
> c.of('C')
'constant'
> c.of('C#')
'upper'
> c.of('SKOS')
'constant'
> c.of('SKOS(-')
'upper'
```

**Comment by IssuehuntBot** [comment]
@sindresorhus has rewarded $36.00 to @chinesedfan. [See it on IssueHunt](https://issuehunt.io/repos/64659785/issues/52)

- :moneybag: Total deposit: $40.00
- :tada: Repository reward(0%): $0.00
- :wrench: Service fee(10%): $4.00
